Scenes/Lab/Lab.tscn

```diff
@@ -3,31 +3,34 @@
 [ext_resource path="res://Scenes/Lab/ButtonContainer.tscn" type="PackedScene" id=1]
 [ext_resource path="res://Scenes/Lab/Lab.gd" type="Script" id=2]
 [ext_resource path="res://Scenes/Lab/Coin.tscn" type="PackedScene" id=3]
-[ext_resource path="res://Assets/Lab/lab-bg.png" type="Texture" id=4]
+[ext_resource path="res://Assets/Lab/lab-bg-sheet.png" type="Texture" id=4]
```

Comment:
Irereject ko yung pr dito ha. Naexperience ko rin to actually kagabi and naayos ko na siya afaik sa merge ko